extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/sql/MSQModeTest.java

```diff
@@ -22,34 +22,36 @@
 import com.google.common.collect.ImmutableMap;
 import org.apache.druid.java.util.common.ISE;
 import org.apache.druid.msq.indexing.error.MSQWarnings;
-import org.apache.druid.query.QueryContext;
 import org.junit.Assert;
 import org.junit.Test;
 
+import java.util.Collections;
+import java.util.Map;
+
 public class MSQModeTest
 {
 
   @Test
   public void testPopulateQueryContextWhenNoSupercedingValuePresent()
   {
-    QueryContext originalQueryContext = new QueryContext();
+    Map<String, Object> originalQueryContext = Collections.emptyMap();
```

Comment:
```suggestion
    Map<String, Object> originalQueryContext = new HashMap();
```

Assigned with `Collections.emptyMap` causes the UT failed.